examples/uv_actor.lua

```diff
@@ -0,0 +1,48 @@
+-- Use LibUV event loop in actor thread
+--
+-- !NOTE! I can not figure out why this does not work
+--  with `inproc` transport so I use `tcp` to communicate
+--  with actor thread
```

Comment:
I do not think its a case. I try add sleep before send and/or recv.
Problem in that `fd` just do not set signaling state.
The only way I found is perform `pipe:recv()` and after that all works.
Also it is does not metter eather you receive message or get `EAGAIN` error.